### DISCUSSION TITLE
Save layout setting for Github Projects

### DISCUSSION THREAD

**Discussion by yogurtsyum** [community/community#9601] [Projects and Issues] [answered]
I think it be nice for the new projects system for Github Teams to have how you want to view it saved, because every time I go to the page it switches from board to table. 

![image](https://user-images.githubusercontent.com/28582329/147992741-5b6181ba-bfc7-4601-9e1d-01e84d669055.png)

P.S. I apologize if this is the wrong category to submit this feedback under.

**Comment by Zackysh** [comment, marked as answer]
I was confused at the beginning also. If any board or table has a blue circle, click on its dropdown, on the bottom you should be able to press "save". I think they should move out the save button or provide a shortcut as it was an editor.

**Reply by yogurtsyum** [reply]
Thanks! This seemed to fix it.

**Reply by Zackysh** [reply]
There are other [issues](https://github.com/github/feedback/discussions/9447#discussion-3773072) dealing with this.
We'll see in the future if github does something about it.
